indigo/release/Dockerfile

```diff
@@ -5,7 +5,6 @@ FROM osrf/ros:indigo-desktop
 MAINTAINER Dave Coleman dave@dav.ee
 
 # apt-commands are combined in single RUN statement with lists folder removal to reduce image size
-RUN apt-get update \
- && apt-get upgrade -y \
- && apt-get install -y ros-indigo-moveit-full \
- && rm -rf /var/lib/apt/lists/*
\ No newline at end of file
+RUN apt-get update && apt-get install -y \
+      ros-${ROS_DISTRO}-moveit-full \
```

Comment:
It's actually set in the official ros core image from the get go: `ENV ROS_DISTRO indigo`

https://github.com/osrf/docker_images/blob/8aed5cc7dc1bcd16b77e5014acc0fe481b6f4468/ros/indigo/indigo-ros-core/Dockerfile#L29